apps/i3wm/i3wm.talon

```diff
@@ -2,82 +2,81 @@
 os: linux
 tag: user.i3wm
 -
-port <number_small>: user.system_command("i3-msg workspace number {number_small}")
-(port flip | flipper): user.system_command("i3-msg workspace back_and_forth")
-port right: user.system_command("i3-msg workspace next")
-port left: user.system_command("i3-msg workspace prev")
+port <number_small>: user.i3wm_switch_to_workspace(number_small)
+(port flip | flipper): user.i3wm_switch_to_workspace("back_and_forth")
+port right: user.i3wm_switch_to_workspace("next")
+port left: user.i3wm_switch_to_workspace("prev")
 
-(win | window) left: user.system_command("i3-msg focus left")
-(win | window) right: user.system_command("i3-msg focus right")
-(win | window) up: user.system_command("i3-msg focus up")
-(win | window) down: user.system_command("i3-msg focus down")
+(win | window) left: user.i3wm_focus("left")
+(win | window) right: user.i3wm_focus("right")
+(win | window) up: user.i3wm_focus("up")
+(win | window) down: user.i3wm_focus("down")
 (win | window) kill: user.system_command("i3-msg kill")
```

Comment:
I think `window close` should also work, right?

Reply:
Yeah, the app.window_close action could be implemented to enable that command. The default of alt-f4 isn't in the default i3 config I think. Happy if you want to implement that action to replace this system_command usage rather than creating an i3 specific one.

Reply:
For some reason I thought that action sent an X11 signal directly rather than pressing keys. I'll override it and call it here.

We can discuss whether `win kill` should be deprecated in a follow-up PR.